src/io-methods/filesystem.js

```diff
@@ -22,12 +24,27 @@ exports.readManifest = function(filePath) {
 }
 
 exports.writeManifest = function(filePath, data) {
-  return new Promise((resolve, reject) => {
+  const jsonPromise = new Promise((resolve, reject) => {
     fs.writeFile(filePath, data, function(err) {
       if (err)
         reject(`Could not write file ${filePath}`)
       else
         resolve()
     })
   })
+
+  const jsPromise = new Promise((resolve, reject) => {
+    if (!config || !config.writeJsFile) {
```

Comment:
writing a js file is opt in. We  don't do it unless `config.writeJsFile` is truthy